src/PaymentSuite/PaymillBundle/Resources/views/Paymill/scripts.html.twig

```diff
@@ -66,26 +104,29 @@
 
         if (error) {
 
-            $('#paymill_payment_error').html(error.apierror);
-            $("#paymill_view_submit").removeAttr("disabled");
+            oPaymill.oErrorContainer.innerHTML = error.apierror;
+            oPaymill.oSubmit.disabled = 'false';
```

Comment:
is it ``'false'`` or ``false`` ? Both are not considerated as valid in HTML5 specification.

Why not remove the attribute instead ?

```js
oPaymil.oSubmit.removeAttribute('disabled'); // submit button is now active
```